lib/xboxrt/libc_extensions/wchar_ext_.c

```diff
@@ -0,0 +1,7 @@
+#include <wchar.h>
+
+int _snwprintf(wchar_t *buffer, size_t count, const wchar_t *format, ...)
```

Comment:
nit: `size_t` should require string.h or something? - same implict style in wchar.h

Reply:
No, `size_t` is supposed to be provided by multiple headers, including `wchar.h`. It currently does this indirectly (and I didn't change this because I think we shouldn't touch PDCLib internals here). A proper upstream `wchar.h` will eventually resolve this.